docs/migration_guide.md

```diff
@@ -44,10 +44,14 @@ When upgrading your existing Habor instance to a newer version, you may need to
     ```
 
 7. Unzip the new Harbor package and change to `./harbor` as the working directory. Configure Harbor by modifying the file `harbor.cfg`, 
+
+7.1 Configure Harbor by modifying the file `harbor.cfg`, 
 you may need to refer to the configuration files you've backed up during step 2. 
 Refer to [Installation & Configuration Guide ](../docs/installation_guide.md) for more information. 
-Since the content and format of `harbor.cfg` may have been changed in the new release, **DO NOT directly copy `harbor.cfg` from previous version of Harbor.**  
+Since the content and format of `harbor.cfg` may have been changed in the new release, **DO NOT directly copy `harbor.cfg` from previous version of Harbor.** 
 
+7.2 Migrate 'harbor.cfg' with upgrade script,
```

Comment:
To assist you in migrating the harbor.cfg file from v0.5.0 to v1.1.0, a script is provided and described as below. For other versions of Harbor, you need to manually migrate the file harbor.cfg.